x-pack/plugins/security_solution/cypress/screens/kibana_navigation.ts

```diff
@@ -29,4 +29,5 @@ export const KIBANA_NAVIGATION_TOGGLE = '[data-test-subj="toggleNavButton"]';
 
 export const SPACES_BUTTON = '[data-test-subj="spacesNavSelector"]';
 
-export const getGoToSpaceMenuItem = (space: string) => `[data-test-subj="${space}-gotoSpace"]`;
+export const getGoToSpaceMenuItem = (space: string) =>
+  `[data-test-subj="${space}-selectableSpaceItem"]`;
```

Comment:
the test subject selector was changed in https://github.com/elastic/kibana/pull/134454 and caused the tests to fail